addons/mail/static/src/components/thread_viewer/thread_viewer.js

```diff
@@ -131,6 +131,7 @@ Object.assign(ThreadViewer, {
         composerAttachmentsDetailsMode: 'auto',
         hasComposer: false,
         hasMessageCheckbox: false,
+        hasScrollAdjust: true,
```

Comment:
thread viewer is just doing a forward to message list, would be better to have the default on it

Reply:
Still not done comment

Reply:
I thought this was solved to avoid having to props.scrollAdjust !== undefined

Reply:
We don't want undefined on the components that are using the prop because indeed boolean should have only 2 values.

However, we do want undefined on the components that are just forwarding it, so that when not giving any value, it can fallback to the default value of the component actually using it.